### ISSUE TITLE
Difficult to view cell with lots of text

### ISSUE THREAD

**Issue by espen** [jakob/Postico#387]
I have a hstore column which is very difficult to view in the sql query. When hovering over the cell and scrolling it will scroll the entire table, not the contents of the cell. The only way to see the contents seems to be select the text then move the cursor to see more of the contents.

Not sure what is the ideal way here. Click and have a larger box appear with the contents? Being able to scroll the contents when hovering?

See example of a hstore column with text not fitting to the cell:

![sql query d3tvk6c6ae0rou makeplans production 2017-05-30 15-13-56](https://cloud.githubusercontent.com/assets/32258/26585011/2bda5cce-454b-11e7-8f74-703aac046946.png)

**Comment by jakob** [comment]
This is a common problem with all fields that contain a lot of data. The tool tips are not really a nice UI.

I think the best way would be to add some kind of button that opens a (scrollable0 popover with the full contents of the cell.

**Comment by espen** [comment]
Yeah, that sounds good. I need to be able to select and copy text, and also edit it easily. A popup textarea would solve that.
